### PR TITLE
#89 Create a link selector component

### PR DESCRIPTION
**Pull request by fkroockmann**
#89

### CODE REVIEW

File: src/tb/component/linkselector/main.js [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+
+                    return restParams;
+                });
+            },
+
+            initDataView: function () {
+                var config = {
+                        css: {
+                            width: "auto",
+                            height: "auto"
```
Comment (ndufreche) [inline src/tb/component/linkselector/main.js]: No css in the javascipt please

File: src/tb/component/popin/main.js [hunk 2]
```diff
@@ -204,18 +204,21 @@ define('tb.component/popin/main', ['tb.core', 'tb.component/popin/PopIn', 'jquer
...
                 if (jQuery('#' + popIn.getId()).length === 0) {
-                    jQuery('#' + manager.containerId).append('<div id="' + popIn.getId() + '"></div>');
-                }
+                    if (null === document.getElementById(manager.containerId)) {
+                        jQuery(manager.toolbarId).append('<div id="' + manager.containerId + '"></div>');
+                    }
 
-                if (popIn.getContent().length || popIn.getContent() !== '') {
-                    jQuery('#' + popIn.getId()).html(popIn.getContent());
+                    if (jQuery('#' + popIn.getId()).length === 0) {
```
Comment (ndufreche) [inline src/tb/component/popin/main.js]: Assign your jQuery('#' + popIn.getId()) to a variable to lighten your script

File: src/tb/component/linkselector/main.js (hunk 1 above)
Comment (fkroockmann) [inline src/tb/component/linkselector/main.js]: It's a config of a data view, not a really css

**Comment by fkroockmann** [comment]
You can merge, thanks :)

**Pull request merged**